toonz/sources/toonz/xsheetviewer.h

```diff
@@ -624,6 +627,11 @@ class XsheetViewer final : public QFrame, public SaveLoadQSettings {
   void drawPredefinedPath(QPainter &p, PredefinedPath which,
                           const CellPosition &pos, optional<QColor> fill,
                           optional<QColor> outline) const;
+
+  void XsheetViewer::drawPredefinedPath(QPainter &p, PredefinedPath which,
+                                        QPoint xy, optional<QColor> fill,
+                                        optional<QColor> outline) const;
```

Comment:
Travis CI claims an error as follows:

```
/home/travis/build/opentoonz/opentoonz/toonz/sources/toonz/xsheetviewer.h:631:22: error: 
      extra qualification on member 'drawPredefinedPath'
  void XsheetViewer::drawPredefinedPath(QPainter &p, PredefinedPath which,
       ~~~~~~~~~~~~~~^
```

Thank you for the great improvement !